internal/annotations/annotations.go

```diff
@@ -0,0 +1,21 @@
+// This package contains annotations that are not exposed to end users and mainly for
+// testing and debugging purposes.
+//
+// Do not rely on these annotations to customize production workload behavior.
+package annotations
+
+// uVM specific annotations
+
+const (
+	// AdditionalRegistryValues specifies additional registry keys and their values to set in the WCOW UVM.
+	// The format is a JSON-encoded string of an array containing [HCS RegistryValue] objects.
+	//
+	// Registry values will be available under `HKEY_LOCAL_MACHINE` root key.
+	//
+	// For example:
+	//
+	//	"[{\"Key\": {\"Hive\": \"System\", \"Name\": \"registry\\key\\path"}, \"Name\": \"ValueName\", \"Type\": \"String\", \"StringValue\": \"value\"}]"
```

Comment:
should this be base64 encoded JSON string instead?

Reply:
I think since its intended for local/manual testing, its better to err on the side of readability
If someone is trying to pass it through the deploy script or something, theyre already violating the intent of the annotation, in which case, theres no need to support that